README.md

```diff
@@ -1,2 +1,36 @@
-# reactive-streams-scalajs
-Reactive Streams bindings for ScalaJS
+# Reactive Streams for Scala.js
+
+The purpose of Reactive Streams is to provide a standard for asynchronous stream processing with non-blocking backpressure.
+
+This repository manages the [Scala.js](http://www.scala-js.org/)
+builds of the API. For details and the TCK see
+[the JVM](https://github.com/reactive-streams/reactive-streams-jvm).
```

Comment:
Technically, concurrency is also a thing for JS, as even though multi threading complicates things, concurrency has more to do with asynchrony.

Porting the TCK would be cool, but I'm unsure about how to proceed. First of all a compatible testing library needs to be chosen, preferably one that works with Scala.js and can handle asynchronous results. When I started toying with this, there were no such libraries, so I [built my own](https://github.com/monixio/minitest). Nowadays I think ScalaTest might work, not sure about asynchronous results though, plus it's bloated, but sort of a standard.

Anyway, the difficulty is that you cannot block threads waiting for the result of asynchronous computations. So you can't just copy/paste from the JVM version. It's doable but I'd prefer for these interfaces to be published first?